File: apps/comic-service-server/src/APIs/Comic/Dtos/Comic.cs
using ComicService.Core.Enums;

namespace ComicService.APIs.Dtos;

public class Comic
{
    public List<string>? Chapters { get; set; }

    public string? Cover { get; set; }

    public DateTime CreatedAt { get; set; }

    public GenresEnum? Genres { get; set; }

    public string Id { get; set; }

    public StatusEnum? Status { get; set; }

    public string? Summary { get; set; }

    public TagsEnum? Tags { get; set; }

    public string? Title { get; set; }

    public DateTime UpdatedAt { get; set; }
}

File: apps/comic-service-node-admin/src/api/comic/Comic.ts
import { Chapter } from "../chapter/Chapter";
import { JsonValue } from "type-fest";

export type Comic = {
  chapters?: Array<Chapter>;
  cover: JsonValue;
  createdAt: Date;
  genres?: Array<"Option1">;
  id: string;
  status?: "Option1" | null;
  summary: string | null;
  tags?: Array<"Option1">;
  title: string | null;
  updatedAt: Date;
};
